docs/index.md

```diff
@@ -0,0 +1,98 @@
+---
+organization: Turbot
+category: ["software development"]
+icon_url: "/images/plugins/turbot/awscfn.svg"
+brand_color: "#FF9900"
+display_name: "AWS CloudFormation"
+short_name: "awscfn"
+description: "Steampipe plugin to query data from AWS CloudFormation template files."
+og_description: "Query AWS CloudFormation template files with SQL! Open source CLI. No DB required."
+og_image: "/images/plugins/turbot/awscfn-social-graphic.png"
+---
+
+# AWS CloudFormation + Steampipe
+
+An AWS CloudFormation template file is used to declare resources, variables, modules, and more.
+
+[Steampipe](https://steampipe.io) is an open source CLI to instantly query data using SQL.
+
+Query all resources in your AWS CloudFormation files:
+
+```sql
+select
+  name,
+  type,
+  jsonb_pretty(properties) as resource_properties
+from
+  awscfn_resource;
+```
+
+```sh
+> select name, type, jsonb_pretty(properties) as args from awscfn_resource;
```

Comment:
I think we can probably remove this line `> select name, type, jsonb_pretty(properties) as args from awscfn_resource;` and just keep this block to the output

Can you please update the example block here to match the query above on lines 22 - 27, and also have it include 2-3 more resources from EC2 and IAM?